projects/go-lib/src/lib/components/go-select/go-select.component.scss

```diff
@@ -141,3 +141,42 @@
 body > .ng-dropdown-panel {
   margin-left: 4px;
 }
+
+.ng-dropdown-panel .ng-dropdown-panel-items .ng-option {
+  background: $theme-light-bg-active;
+  font-size: .875rem;
+
+  &.ng-option-marked,
+  &.ng-option-selected,
+  &.ng-option-selected.ng-option-marked {
+    background-color: $theme-light-bg-hover;
+
+    .ng-option-label {
+      font-weight: $weight-light;
+    }
+  }
+}
+
+.ng-dropdown-panel .ng-dropdown-header {
+  padding: 0;
+}
+
+.go-select__select-all-button {
+  @include transition(all);
+
+  background: $theme-light-bg-active;
```

Comment:
If what I said above is correct, and based on the image in the DSM, it seems like the background of this should be white rather than gray, no?